scripts/vscripts/statcollection/lib/statcollection.lua

```diff
@@ -364,8 +376,10 @@ function statCollection:sendStage3(winners, lastRound)
     -- Build players array
     local players = {}
     for i = 1, (PlayerResource:GetPlayerCount() or 1) do
+        local steamID = PlayerResource:GetSteamAccountID(i - 1)
```

Comment:
What's the point in putting a variable for this?

Where you planning on doing a check on the ID or something?